doc/releases/changes.rst

```diff
@@ -5,6 +5,8 @@ Release Notes
 
 Version 5.2.0 (unreleased month date, year)
 -------------------------------------------
+- Update Vega-Lite from version 5.15.1 to version 5.16.3;
+  see `Vega-Lite 5.16.3 Release Notes <https://github.com/vega/vega-lite/releases/tag/v5.16.3>`_.
```

Comment:
Should we link here just to the general release notes page (https://github.com/vega/vega-lite/releases) as it's more then just the 5.16.3 changes?